src/DynamoCoreWpf/ViewModels/Search/ClusterAutocompleteResult.cs

```diff
@@ -0,0 +1,7 @@
+namespace Dynamo.Wpf.ViewModels
+{
+    public class ClusterAutocompleteResult
+    {
+        public string Description { get; set; }
```

Comment:
Thought we already have a data class for the result?

Reply:
Yes but it's not accesible, so i opted to do this instead of changing access modifiers of some of the data classes so we can keep consistency there or not make those classes public.

Reply:
Maybe the class itself can be `internal`?

Reply:
`NodeAutoCompleteSearchViewModel` `IEnumerable<ClusterAutocompleteResult> ClusterResults` needs to be accessible from the WPF framework, so it needs to be public. So the class ClusterAutocompleteResult needs to be public too, else it throws an incoherent access modifiers compilation error.